Mathlib/MeasureTheory/Constructions/Polish.lean

```diff
@@ -634,6 +634,11 @@ instance Quotient.borelSpace {X : Type*} [TopologicalSpace X] [PolishSpace X] [M
   ⟨continuous_quotient_mk'.map_eq_borel (surjective_quotient_mk' _)⟩
 #align quotient.borel_space Quotient.borelSpace
 
+@[to_additive AddCosetSpace.borelSpace]
+instance CosetSpace.borelSpace {G : Type*} [TopologicalSpace G] [PolishSpace G] [Group G]
```

Comment:
Can you add a comment explaining how this differs from `QuotientGroup.borelSpace` immediately below? Is one a special case of the other?

Also, I'm not sure if there is any precedent for `CosetSpace` as a name.

Reply:
It turns out that a number of long-standing names, e.g., [QuotientGroup.continuous_smul₁](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Group/Basic.html#QuotientGroup.continuous_smul%E2%82%81), don't assume that `Γ` is normal, and so `QuotientGroup` is a misnomer (`CosetSpace` would be more accurate). But we can go for consistency over accuracy if you prefer?...

Reply:
Moreover, if we erase this `CosetSpace.borelSpace` instance, then typeclass inference seems to fail, and, e.g., `MeasurableSMul G (G ⧸ Γ)` fails to synthesize. By definition, `G ⧸ Γ` is the quotient of `G` by the action of `Γ`, but it seems unable to pick up the `BorelSpace` instance quotient unless we restate it.

Reply:
done